ops/testing.py

```diff
@@ -1546,7 +1546,14 @@ def make_dir(
             )
 
     def remove_path(self, path: str, *, recursive: bool = False):
-        file_or_dir = self._fs.get_path(path)
+        try:
+            file_or_dir = self._fs.get_path(path)
+        except FileNotFoundError:
+            if recursive:
+                return
```

Comment:
Nit: would be nice to include the "why" comment I showed earlier, for example `# Pebble doesn't give not-found error when recursive is specified`.